src/guides/v2.3/frontend-dev-guide/layouts/product-layouts.md

```diff
@@ -21,6 +21,24 @@ Layout file | Description
 `catalog_product_view_type_simple.xml` | Layout from this file is applied to `simple` product only
 `catalog_product_view_type_virtual.xml` | Layout from this file is applied to `virtual` product only
 
+## Customize Product view page
+
+With the help gives numerous containers on the product page, You should simply reference the container and add blocks to it.
+
+*  `product.info.main`
+*  `product.info.price`
+*  `product.info.stock.sku`
+*  `product.info.form.content`
+*  `product.info.extrahint`
+*  `product.info.social`
+*  `product.info.media`
+
+Sample of usage in the product page layout containers:
+
```

Comment:
Please describe what the example does.

Reply:
In this case, the sample code seems to be simple and intuitive, so it doesn't need the explanation of the result. Be mindful of future contributions however. More complex code samples usually require this type of explanation.